runtime/src/lib.rs

```diff
@@ -398,16 +389,16 @@ impl orml_tokens::Trait for Runtime {
 #[derive(Debug)]
 pub struct OffchainAppCrypto;
 
-impl frame_system::offchain::AppCrypto<MultiSigner, MultiSignature> for OffchainAppCrypto {
-    type GenericPublic = sp_runtime::app_crypto::sr25519::Public;
-    type GenericSignature = sp_runtime::app_crypto::sr25519::Signature;
+impl frame_system::offchain::AppCrypto<AccountId, Signature> for OffchainAppCrypto {
```

Comment:
The `MultiSignature` enum came from the official template and I removed it because: (1) The pallet runtime (`sr25519`) was working fine while the RPC runtime (`MultiSignature`) was not. (2) It was easier to satisfy all trait constraints of all different runtimes using `sr25519` and (3) We currently don't require different types of signature (`sr25519`, `ed25519` or `ecdsa`).

If desirable, I can try to reintroduce `MultiSignature` in a following PR

Reply:
I don't have enough knowledge to know if we would like to reintroduce it and what difference would it make for the offchain worker keys. If you see a compelling usage we can create a ticket to be able to track it an later re-introduce it otherwise let's leave it like that.